### DISCUSSION TITLE
Error when building app

### DISCUSSION THREAD

**Discussion by MaximeCoteL** [RailFansCanada/RailFansMap#297] [Q&A]
Cannot read properties of undefined (reading 'D:\Documents\transport_en_commun\RailFansMap\.pnp.cjs

![image](https://github.com/RailFansCanada/RailFansMap/assets/54948898/3f42a8ee-d216-4688-b614-b4dc5482f701)

Didn't work on my fantasy lines in a while and haven't changed anything, and now I get yarn errors when starting... Any ideas what the error mean ? I tried changing versions in package.json but wasn't able to fix it.

**Comment by dellisd** [comment]
Everything is building normally on my end (with the same version of yarn) but here are a few things to check first:

1. What version of Node are you using?
2. You could try deleting the `.pnp.cjs` file and let yarn regenerate the file.
3. Have you added any new dependencies in your fork that aren't in the main project?

**Comment by MaximeCoteL** [comment]
1. I am using node v18.18.2
2. I tried and it didn't fix anything.
3. I have not !

Finally, I'm not certain if I changed anything compared to last time I tried to start it, but now it starts, but when I ctrl + click on http://localhost:8080/ to open it, I get that last line ''wait until bundle finished'' but it's stuck there..
![image](https://github.com/RailFansCanada/RailFansMap/assets/54948898/7a96c6b0-2f9e-49fc-8456-9606801c4e7b)
